inc/audit_log_to_cloudwatch/class-cloudwatch-driver.php

```diff
@@ -125,7 +127,12 @@ public function get_records( $args ) {
 			'queryString'    => $query,
 		];
 
-		$query = cloudwatch_logs_client()->startQuery( $params );
+		try {
+			$query = cloudwatch_logs_client()->startQuery( $params );
```

Comment:
This should be caught by the caller, and error displayed there. This function should continue to Throw (or return a WP_Error). Calling the function `get_records` should not output an error to the output buffer, as this function could be called in other contexts.

Reply:
I looked into stream plugin code and it was not checking WP_Error or catching and throwable exception.
Reference : https://github.com/xwp/stream/blob/e207cf8795542eb73bdce7b80f037ea8e3a22e55/classes/class-db.php#L203

Also, there is no way to filter the list table `no_items` function. 

https://github.com/xwp/stream/blob/e207cf8795542eb73bdce7b80f037ea8e3a22e55/classes/class-list-table.php#L80

Reply:
Hmm ok- I wonder if we might be better off trigger an error in the error log then. I think the output for this function call still has to be removed. We could still hook in in admin notices or similar to show an error that we store elsewhere in memory

Reply:
Ping @faishal

Reply:
Done, I stored in static variable of the class and moved rendering part on `wp_stream_after_list_table`